mkdocs/docs/HPC/running_batch_jobs.md

```diff
@@ -803,6 +803,28 @@ The output of the various commands interacting with jobs (`qsub`,
 `stat`, ...) all depend on which `cluster` module is loaded.
 {% endif %}
 
+### cross cluster submission
+
+We made it possible to submit to a cluster different from the cluster that your current worker node is on. 
+This could come in handy if, for example, the submit command only works on the current clusters worker nodes, but the job itself can run on other clusters. An example of this is the `wsub` command of `worker`. More technical info why the `wsub` command can only be executed on the {{defaultcluster}} cluster can be found [here](troubleshooting.md#multi-job-submissions-on-a-non-default-cluster).
+
+```shell
+{% include "examples/Multi-job-submission/par_sweep/weather.pbs" %}
+```
+
+As mentioned before, the `wsub` command will only work on the `{{defaultcluster}}` cluster. But this does not mean that the job itself can't be executed on other clusters. For example if you want to submit to `{{othercluster}}` cluster, you can only change your submit cluster by using `module swap env/surm/{{othercluster}}` instead of `module swap cluster/{{othercluster}}`. The wsub command will then be executed on a {{defaultcluster}} worker node, but the jobs itself will be submitted to the `{{othercluster}}` cluster. It is ofcourse also possible to use an other cluster then `{{othercluster}}`.
+
+
+We didn't only create the `env/slurm/{{defaultcluster}}` part, but we actually split every cluster in 3 different parts. Here illustrated with the `{{defaultcluster}}` cluster, but this can be changed to every cluster you like.
```

Comment:
```suggestion
Each ``cluster`` module not only loads the corresponding ``env/slurm/...`` module to control where jobs are sent to, but also tow other ``env/...`` modules which control other parts of the environment.
For example, for the `{{defaultcluster}}` cluster:

We didn't only create the `env/slurm/{{defaultcluster}}` part, but we actually split every cluster in 3 different parts. Here illustrated with the `{{defaultcluster}}` cluster, but this can be changed to every cluster you like.